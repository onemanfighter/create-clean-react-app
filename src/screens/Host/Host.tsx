import { Outlet } from 'react-router-dom';
import { Box } from '@chakra-ui/react';

import { CopyrightFooter } from './CopyrightFooter';

const Host = () => {
  return (
    <Box minH={'100vh'}>
      <Outlet />
      <CopyrightFooter />
    </Box>
  );
};

export default Host;
