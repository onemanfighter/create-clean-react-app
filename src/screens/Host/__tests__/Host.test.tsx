import Host from '../Host';
import { renderWithRouter } from '@testUtils';

describe('Host', () => {
  it('should render correctly', () => {
    const { container } = renderWithRouter(<Host />);

    expect(container).toMatchSnapshot();
  });
});
