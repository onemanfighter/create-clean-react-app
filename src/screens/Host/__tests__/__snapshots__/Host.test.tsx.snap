// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Host should render correctly 1`] = `
<div>
  <div
    class="css-vooagt"
  >
    <div
      class="css-o5g2qp"
    >
      <p
        class="chakra-text css-xi606m"
      >
        Â© 2024 Template-boilerplate | All rights reserved
      </p>
    </div>
  </div>
</div>
`;
